src/olympia/addons/serializers.py

```diff
@@ -87,16 +87,23 @@ class Meta:
         fields = ('name', 'text', 'url')
 
 
+class CompactLicenseSerializer(LicenseSerializer):
+    class Meta:
+        model = License
+        fields = ('name', 'url')
```

Comment:
I'm a fan of exposing the `id` by default too, not sure what @diox thinks about that but I found it incredibly helpful for debugging or tracing things.